R/player_transfer_history.R

```diff
@@ -12,7 +12,7 @@
 #'
 #' @export
 #'
-tm_player_transfer_history <- function(player_urls) {
+tm_player_transfer_history <- function(player_urls, get_extra_info = TRUE) {
```

Comment:
You'll also need to add  an additional `@param` for `get_extra_info` to the roxygen notes, otherwise the check will throw an error or warning